uportal-war/src/main/java/org/apereo/portal/portlets/lookup/PersonLookupHelperImpl.java

```diff
@@ -440,19 +440,44 @@ protected IAuthorizationPrincipal getPrincipalForUser(final IPerson person) {
     }
 
     /**
+     * Filter the provided set of user attribute names to contain only those the specified principal
+     * has permissions to view.
+     *
+     * @param principal
+     * @param generallyPermittedAttributes
```

Comment:
I don't offhand know what `generallyPermitted` means in this context. Suggest using JavaDoc comment on the parameter to explain.